CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # RELEASE NOTES
 
+## [0.4.0] - 2020-01-25
+
+### Added
+
+- Ability to disable ingress route creation
```

Comment:
We also added the ability to specify per-service middleware injection for Traefik2 IngressRoutes